RUFAS/routines/animal/animal_combinations.py

```diff
@@ -1,2 +1,13 @@
-# TODO: In a separate PR, move the AnimalCombinations inside Pen class
-#  here to avoid circular imports.
+from enum import Enum
+
+
+class AnimalCombination(Enum):
+    """
+    Enumeration that represents the valid combinations of animals in a pen.
+    """
+    CALF = 0  # calves
+    GROWING = 1  # heiferIs, heiferIIs
+    CLOSE_UP = 2  # heiferIIIs, dry cows
+    LAC_COW = 3  # lactating cows
```

Comment:
```suggestion
    Enumeration that represents the valid combinations of animals in a pen.
    
    Attributes
    ----------
    CALF: int
        Represents a calf-only pen (I think).
    ...
    
    """
    CALF = 0 
    GROWING = 1
    CLOSE_UP = 2
    LAC_COW = 3
```
I think this would be a good opportunity to add a proper docstring to the class.

Reply:
Sounds good thank you!